parachains/common/kusama/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use core::marker::PhantomData;
+use frame_support::traits::Contains;
+use sp_core::Get;
+use xcm::latest::prelude::*;
+
+pub struct RelayOrOtherSystemParachains<Runtime: parachain_info::Config> {
```

Comment:
what about replacing this:
`Runtime: parachain_info::Config`
with this:
`SelfParaId: Get<ParaId>`

and then use it like:
```
kusama_common::RelayOrOtherSystemParachains<parachain_info::Pallet<Runtime>>
```

at least we dont need `parachain-info` dependency anywhere here

Reply:
It did cross my mind, but potentially they could put another ParaId in that way. By taking a parachain_info there's no possible room for mistake.